### ISSUE TITLE
Audio won't play

### ISSUE THREAD

**Issue by djmoffat** [BrechtDeMan/WebAudioEvaluationTool#226]
Hey,

I have been having some issues that result in the test breaking. I clicked on a few sliders on the APE interface and the audio player broke.

First error was 
`core.js:1978 Uncaught TypeError: Failed to execute 'linearRampToValueAtTime' on 'AudioParam': The provided float value is non-finite.`

Then the consistent error of 
`core.js:1978 Uncaught TypeError: Failed to execute 'linearRampToValueAtTime' on 'AudioParam': The provided float value is non-finite.`

Which is breaking the test and not passing audio. Let me know if you know anything or need anything else

**Comment by nickjillings** [comment]
Potential fix in 30440db (hot_fix).

MWE below:
```xml
<?xml version="1.0" encoding="utf-8"?>
    <waet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="test-schema.xsd">
        <setup interface="APE" projectReturn="save.php" randomiseOrder="false" poolSize="2" loudness="-23" playOne="false" crossFade="0.5">
            <metric>
                <metricenable>testTimer</metricenable>
            </metric>
            <interface/>
        </setup>
        <page id='test-0' hostURL="media/example/" randomiseOrder='true' repeatCount='0' loop='false' loudness="-12">
            <commentboxprefix>Comment on fragment</commentboxprefix>
            <interface name="test">
                <title>MWE</title>
                <scales>
                    <scalelabel position="0">Bad</scalelabel>
                    <scalelabel position="25">Poor</scalelabel>
                    <scalelabel position="50">Fair</scalelabel>
                    <scalelabel position="75">Good</scalelabel>
                    <scalelabel position="100">Excellent</scalelabel>
                </scales>
            </interface>
            <audioelement url="0.wav" id="track-0" type="outside-reference" />
            <audioelement url="1.wav" id="track-1" />
            <audioelement url="2.wav" id="track-2" />
        </page>
    </waet>

```

**Comment by djmoffat** [comment]
The issue is based around audio normalisation if the sample length is less than 0.5s long - there is not enough duration to calculate loudness, and so gain calculation is Inf - which is not a number, so not a valid input gain

**Comment by nickjillings** [comment]
Loudness normalisation buffer now ensures the minimum sample size is 0.4s long and must always return at least one frame of information.

Fixed in 4d6ca22
